<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTranslateDialog</class>
 <widget class="QDialog" name="AddTranslateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add translate</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Word:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditWord"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Translate:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditTranslate"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Part of Speech</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QRadioButton" name="rbVerb">
                <property name="text">
                 <string>Verb</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rbNoun">
                <property name="text">
                 <string>Noun</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rbAdjective">
                <property name="text">
                 <string>Adjective</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rbAdverb">
                <property name="text">
                 <string>Adverb</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QRadioButton" name="rbPronoun">
                <property name="text">
                 <string>Pronoun</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rbPreposition">
                <property name="text">
                 <string>Preposition</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rbConjuction">
                <property name="text">
                 <string>Conjuction</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rbInterjection">
                <property name="text">
                 <string>Interjection</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder></zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddTranslateDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddTranslateDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
